### ISSUE TITLE
Improve non-normative command descriptions and examples

### ISSUE THREAD

**Issue by andreastt** [w3c/webdriver#1233]
Many people have requested that we explain in layman terms what commands do and how they interact with eachother. In #1105 and #1231 we removed or modified command introductions into notes (like HTML has) to avoid confusion over normative introductory language.

This issue will track improving comamnd descriptions and their examples so the specification will be easier to read for client implementors and users.

**Comment by jlipps** [comment]
great idea :-)
